extensions/Worker.Extensions.Http.AspNetCore/src/Coordinator/DefaultHttpCoordinator.cs

```diff
@@ -5,24 +5,40 @@
 using System.Collections.Concurrent;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore.Infrastructure;
 
 namespace Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore
 {
     internal class DefaultHttpCoordinator : IHttpCoordinator
     {
+        private const int HttpContextTimeoutSeconds = 5;
+        private const int FunctionContextTimeoutSeconds = 5;
+        private const int FunctionRunTimeoutSeconds = 15;
```

Comment:
Would it make more sense to name this `FunctionStartTimeoutSeconds` or something similar? My first though reading this was a timeout for how long the function runs, but it's actually used with `FunctionStartTask`.

Reply:
Yeah, I like the name. The method exposed by the coordinator is called `RunFunctionInvocationAsync`, so I ended up deriving from that, but what you're suggesting would make this clearer.